administrator/components/com_content/View/Article/HtmlView.php

```diff
@@ -78,6 +79,18 @@ public function display($tpl = null)
 		$this->state = $this->get('State');
 		$this->canDo = \JHelperContent::getActions('com_content', 'article', $this->item->id);
 
+		$params = ComponentHelper::getComponent('com_associations')->getParams();
+		$this->set('remember', $params->get('remember', '0', 'cmd'));
+
+		if ($this->get('remember') == '1')
+		{
+			$this->set('decision', $params->get('decision'));
+		}
+		else
+		{
+			$this->set('decision', '');
+		}
```

Comment:
No need for the else statement, just do `$this->set('decision', '');` before the previous IF, also lots of uncommented code, keep it tidy 🙂